provisioning_client/inc/azure_prov_client/prov_device_client.h

```diff
@@ -32,6 +32,8 @@ extern "C"
 {
 #endif
 
+static const char* const PROV_OPTION_DO_WORK_FREQUENCY_IN_MS = "do_work_freq_ms";
```

Comment:
If I understand correctly, if this is static, it will create an instantiation of this every time this is included (which could be wasteful from a memory perspective). Should this drop the static?

Reply:
We need the static since it's being declared in .h.  If you look at `prov_device_client_ll.h` and `iothub_client_options.h` this is the pattern they use.  Basically - iothub_client_options.h has e.g. below.  I'm adding the STATIC_VAR_UNUSED since to prov options since it's just a matter of time before something trips up over that anyway.

```c
static STATIC_VAR_UNUSED const char* OPTION_CONNECTION_TIMEOUT = "connect_timeout";
```